packages/api-clients/open-api/tenantApi.yml

```diff
@@ -855,8 +855,15 @@ paths:
       operationId: revokeCertifiedAttributeById
       description: Revoke a certified attribute to a Tenant by the requester Tenant
       responses:
-        "204":
-          description: Certified Attribute revoked
+        headers:
+          X-Metadata-Version:
+            $ref: "#/components/headers/MetadataVersionHeader"
```

Comment:
Missing declaration in `components`, see: https://github.com/pagopa/interop-be-monorepo/pull/1848/files#diff-a50f62f3a4939e0ad8d4f443bf76522090263ad0bfabf14e174d8d83475fc372R406

Reply:
Added [here](https://github.com/pagopa/interop-be-monorepo/commit/46cd17a10931b7a0e9cc7ee1d813cb47c406c842)